requirements.txt

```diff
@@ -11,4 +11,6 @@ pytest==7.4.0
 setuptools
 pandas-stubs
 types-psycopg2
-types-requests
\ No newline at end of file
+types-requests
+moralis
```

Comment:
The requirements file is a bit inconsistent with respect to pinning versions. I would generally say that tools purely for development (`black`, `pylint`, `mypy`, `pytest`) should not be fixed strictly while tools core to the functionality (most of the rest) should be pinned (or at least restricted on the major or minor version).